chalice/app.py

```diff
@@ -317,12 +349,30 @@ def __eq__(self, other):
         return self.__dict__ == other.__dict__
 
 
+class APIGateway(object):
+
+    _DEFAULT_BINARY_TYPES = [
```

Comment:
We should evaluate some of the common binary types.  I realize there's a limit, but I think we should add more common types here besides images, especially now that customers can override the list if they want complete control.